example/example-MapValuesMultipleBigMaps copy.ts

```diff
@@ -0,0 +1,69 @@
+import { MichelsonMap, TezosToolkit } from '@taquito/taquito';
```

Comment:
This file looks to be the same as example/example-MapValuesMultipleBigMaps.ts

Reply:
whooops!  fixed